spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/utils/ConnectorUtil.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark.connector.utils;
+
+import static com.datastrato.gravitino.spark.connector.ConnectorConstants.COMMA;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+
+public class ConnectorUtil {
+
+  public static String removeDuplicates(String[] elements, String otherElements) {
+    Set<String> uniqueElements = new LinkedHashSet<>(Arrays.asList(elements));
+    if (StringUtils.isNotBlank(otherElements)) {
+      uniqueElements.addAll(Arrays.asList(otherElements.split(COMMA)));
```

Comment:
if otherElements may contains multi element ,please pass a array not string as method parameters

Reply:
> Contributor

ok

Reply:
updated.